### PR TITLE
OFFAPPS-1084 / Remove space under app

### PR DESCRIPTION
**Pull request by keitheous**
Remove Space Under App(Under buttons)

## Description
Sometimes on page load, the app resizes and pushes the content of the app up. After further investigation. We realised that it is cause by the drop down menu that is hidden in the background. However it is not exactly hidden. This only occurs in a language with long texts. 

## Tasks
- [x] npm run test & lint
- [x] overwrite `.c-menu` to display none

## References
[JIRA ticket](https://zendesk.atlassian.net/browse/OFFAPPS-1084)

## Screenshots (if needed)
<details>
  <summary>Before</summary>
  screen shot 2018-11-12 at 2 43 21 pm
</details>

<details>
  <summary>After</summary>
screen shot 2018-11-12 at 2 42 03 pm
</details>

## CCs
@zendesk/apps-migration
@yu-tian113 

## Risks
* [low] change css - really hide the tray when it is meant to be hidden.

### CODE REVIEW

**Comment by vaevictis** [comment]
Talk about a _well hidden edge case..._

File: src/main.scss [hunk 1]
```diff
@@ -50,6 +50,10 @@ textarea.message {
 }
 
 .datalist[data-id="groups"] {
+  .c-menu[aria-hidden=true] {
```
Comment (vaevictis) [inline src/main.scss]: Not too sure what are the consequences of using `aria-hidden`. I see why this approach would solve the bug, but I wonder if it's semantically reliable. `aria-` attributes are normally designed for accessibility purposes right?

Comment (ocke) [reply src/main.scss]: I think that was there from the v1 version of the app, so I kept it in without thinking about it.

Comment (yu-tian113) [reply src/main.scss]: @vaevictis, this is from garden css, aria-hidden element is set to be `visibility: hidden` which may cause some issue if you want to physically remove the element from document flow(display:none).
I get your concern is whether we should use aria attribute to control UI, I'm not 100% sure, but personally I don't think it's a bad practice as long as we don't make it confusing.
Valid point though!

**Comment by keitheous** [comment]
Hi, @ocke . I tested this on ie11 and it worked. 

You were also right about uploading this after manually zipping dist directory using `npm run build` instead of `npm run build:dev` / `npm run watch`.

ie1test

**Pull request merged**